mtp_api/urls.py

```diff
@@ -47,4 +43,6 @@
         _('Sorry, something went wrong'),
         content_type='text/plain', status=500,
     )),
+
+    url(r'^$', lambda request: HttpResponse(content_type='text/plain', status=204)),
```

Comment:
Any reason for moving this down into a separate block?

Reply:
it seemed more like an error page / fallback than important app code. it fits less well with the group of includes at the top